### ISSUE TITLE
Syntax error on sass_extensions.rb at line 149

### ISSUE THREAD

**Issue by rromerogar** [thomasbachem/compass-svg-templates#1]
```
$ compass compile                               
SyntaxError on line ["54"] of /home/rromero/.rvm/rubies/ruby-1.9.3-p484/lib/ruby/site_ruby/1.9.1/rubygems/core_ext/kernel_require.rb: /home/rromero/.rvm/gems/ruby-1.9.3-p484/gems/compass-svg-templates-1.0.1/lib/svg-templates/sass_extensions.rb:149: invalid multibyte char (US-ASCII)
/home/rromero/.rvm/gems/ruby-1.9.3-p484/gems/compass-svg-templates-1.0.1/lib/svg-templates/sass_extensions.rb:149: invalid multibyte char (US-ASCII)
/home/rromero/.rvm/gems/ruby-1.9.3-p484/gems/compass-svg-templates-1.0.1/lib/svg-templates/sass_extensions.rb:149: syntax error, unexpected $end, expecting keyword_end
...al_path} cannot be processed – probably invalid?"
...                               ^
Run with --trace to see the full backtrace
```

The problem is the caracter **–** in this file. Works fine if changet with **-**.

**Comment by thomasbachem** [comment]
Could you provide the problematic SVG file?

**Comment by rromerogar** [comment]
The problematic file is sass_extensions.rb in your extension.The problem is the caracter – in sass_extensions.rb at line 149.

'''
/home/rromero/.rvm/gems/ruby-1.9.3-p484/gems/compass-svg-templates-1.0.1/lib/svg-templates/sass_extensions.rb:149: invalid multibyte char (US-ASCII)
'''

**Comment by rromerogar** [comment]
The problem is in this line:

raise Compass::Error, "SVG file #{real_path} cannot be processed – probably invalid?"

The character between processed and probably.

**Comment by thomasbachem** [comment]
Oh, hm, ok. Thanks for reporting! I changed it to an ASCII minus character and released v1.0.2.
